syntax = "proto3";

package test;
option go_package = "./;test";

message HelloRequest {
    string id = 1;
    string msg = 2;
}

message HelloRequest2 {
    string id = 1;
    string msg = 2;
}

message HelloResponse {
    string id = 1;
    string msg = 2;
}

service HelloService {
    rpc Hello(HelloRequest) returns (HelloResponse);
    rpc HelloStreamA(stream HelloRequest) returns (HelloResponse);
    rpc HelloStreamB(HelloRequest) returns (stream HelloResponse);
    rpc HelloStreamC(stream HelloRequest) returns (stream HelloResponse);
}
